### ISSUE TITLE
[Issue]: Collections thumbnail should respect parental guidance settings for user

### ISSUE THREAD

**Issue by csaavedra** [jellyfin/jellyfin#10195]
### Please describe your bug

I have added a user with parental guidance restriction set to TV-Y7. When opening a client with that user, Jellyfin correctly shows only the shows and movies that are rated TV-Y7 or less. However, the Collection thumbnail (the backdrop under "Movies", "Series", etc) is showing content that is MA or PG. Of course, those shows/movies are not visible, but the image used there should only come from media that is allowed for the user.

### Jellyfin Version

10.8.z

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SamJongenelen** [comment]
I am not yet familiar enough with JF to know for sure, but it looks like [this](https://github.com/jellyfin/jellyfin/blob/1141883f7734126aefc1d33172261a278c3ada84/Emby.Server.Implementations/Library/SplashscreenPostScanTask.cs#L70C17-L70C17) line is a todo relevant. It now uses a hardcoded `MaxParentalRating` of 10.

This splashscreen code is a background process triggered by the `LibraryManager`, so it has no context of a User when ran. I guess this is why the TODO is left its no oneline change.

**Comment by csaavedra** [comment]
That sounds like a workaround, dunno. I'm only a user of Jellyfin, so not an authoritative figure around here in any way, but in my experience working on other projects just adding a setting to work around a limitation is usually a poor choice, as these tend to pile up. I'd advocate for making sure that the splashscreen/library manager are user-aware.

**Comment by srcrist** [comment]
Closing this issue as a duplicate of https://github.com/jellyfin/jellyfin/issues/10116. These images are actually pre-generated and are not generated on a per-user basis at all. While waiting on such a feature to be added, if ever, server administrators can always set a static image for the library that is not generated by the collage process at all, if they wish.
